server/lobbyconnection.py

```diff
@@ -530,14 +539,75 @@ async def check_policy_conformity(self, player_id, uid_hash, session, ignore_res
 
         return response.get('result', '') == 'honest'
 
+    async def command_auth(self, message):
+        pub_key = config._API_JWT_PUBLIC_KEY_VALUE
+        token = message["token"]
+        unique_id = message['unique_id']
+
+        try:
+            token = jwt.decode(token, pub_key, algorithms="RS256")
+            player_id = token["user_id"]
+        except (InvalidTokenError, KeyError):
+            raise AuthenticationError("Token signature was invalid")
+
+        new_irc_password = b64encode(os.urandom(30)).decode()
```

Comment:
I'm not super familiar with IRC, but it might be possible that a Base64 encoded String contains control commands. Maybe go for sha256 or random UUID instead?

Reply:
Oh because of the `/` character.